Plutarch/Api/V1/Value.hs

```diff
@@ -116,6 +119,26 @@ instance PEq (PValue 'Sorted 'Positive) where
 instance PEq (PValue 'Sorted 'NonZero) where
   a #== b = pto a #== pto b
 
+{- | Partial ordering implementation for sorted 'PValue' with 'Positive' amounts.
+
+Use 'pcheckBinRel' if 'AmountGuarantees' is 'NoGuarantees'.
+-}
+instance POrd (PValue 'Sorted 'Positive) where
+  a #< b = pforgetPositive @_ @'Sorted @'NonZero a #< pforgetPositive b
+  a #<= b = pforgetPositive @_ @'Sorted @'NonZero a #<= pforgetPositive b
+
+{- | Partial ordering implementation for sorted 'PValue' with 'NonZero' amounts.
+
+Use 'pcheckBinRel' if 'AmountGuarantees' is 'NoGuarantees'.
+-}
+instance POrd (PValue 'Sorted 'NonZero) where
+  a #< b = f # a # b
+    where
+      f = phoistAcyclic $ pcheckBinRel #$ phoistAcyclic $ plam (\x y -> pfromData x #< pfromData y)
+  a #<= b = f # a # b
+    where
+      f = phoistAcyclic $ pcheckBinRel #$ phoistAcyclic $ plam (\x y -> pfromData x #<= pfromData y)
```

Comment:
I added `POrd` instances but I don't know if these should be standalone `ple` and `pleq` functions instead.

Reply:
It's better this way.